feature/sessions/README.md

```diff
@@ -190,18 +190,18 @@ install(Sessions) {
 
 Alongside SessionStorage there is a `SessionStorageMemory` class that you can use for development.
 It is a simple implementation that will keep sessions in-memory, thus all the sessions are dropped
-once you shutdown the server and will constantly grow in memory since the implementation do not discard
-old sessions at all.
+once you shutdown the server and will constantly grow in memory since this implementation does not discard
+the old sessions at all.
 
 This implementation is not intended for production environments.
 
 #### directorySessionStorage
 
 As part of the `io.ktor:ktor-server-sessions` artifact, there is a `directorySessionStorage` function
-that exposes a session storage that will use a folder for storing sessions on disk.
+which utalizes a session storage that will use a folder for storing sessions on disk.
 
 This function has a first argument of type `File` that is the folder that will store sessions (it will be created
-if doesn't exist already).
+if it doesn't exist already).
 
 Also there is a `cache` optional argument that when set, will keep a 60-second in-memory cache to prevent
 calling the OS from reading the session from disk at all.
```

Comment:
@B1ggDave : directorySessionStorage is a function that has a boolean (true or false) argument called cache (which is optional). When set to true, the function changes its behaviour to keep an in-memory cache. That cache, during 60 seconds, keeps that data in memory, so it doesn't have to read that data from disk; reading for disk means that it will call the OS (operating system) to retrieve the data, and that's usually a costly operation.

Reply:
@soywizI can't work out now how to edit this doc further... I will read up about it now.

There is also an optional `cache` argument, which when set, will keep a 60-second in-memory cache to prevent calling the OS and reading the session from disk each time.